app/models/concerns/post.rb

```diff
@@ -0,0 +1,61 @@
+module Post
+  extend ActiveSupport::Concern
+
+  include NameFromTitle
+  include Slug
```

Comment:
I've changed it back to `Name` for now.  I think that's the simplest option, but it's easy to change later.